test/dotnet-trx-merge.IntegrationTests/DotNetTrxMergeTests.cs

```diff
@@ -112,8 +112,28 @@ public async Task DotnetTrxMerge_WithOutputFolder_Success()
         text.Should().Contain("testId=\"86e2b6e4-df7a-e4fa-006e-c056c908e219\"");
         text.Should().Contain("testName=\"SecondSimpleNumberCompare\"");
     }
-    
-    
+
+    [Fact]
+    public async Task DotnetTrxMerge_WithMultipleFilesFromOurTestRun()
```

Comment:
this test fails, and is not needed, I was just using it to verify that my changes resulted in being able to load the files

Reply:
@ricardofslp this want many to be merged before this was removed, as the test fails. I'll get a pr together to remove this. 

I only added this to show a real example of the data from the wild. I'm not at a computer now, but as soon as I can be ill get a pr together to remove this test and the large data file

Reply:
No worries. I've removed in a PR I did before merging to main.
Please check after if something else should be removed and open a PR if it should